src/sdk-reference/js/6/kuzzle/use-controller/snippets/use-controller.js

```diff
@@ -0,0 +1,13 @@
+class TaxiController extends BaseController {
+  constructor (kuzzle) {
+    super(kuzzle, 'my-plugin/taxi');
+  }
+
+  enroll () {
+    return 'Success';
```

Comment:
should use "this.query" instead

This probably means a custom template with either a mocked version of Kuzzle.query, or more simply a dedicated try...catch returning a success response if the error is _exactly_ about a missing API method, paving the way for more thorough end-to-end tests with plugins.